.pylintrc

```diff
@@ -0,0 +1,425 @@
+[MASTER]
```

Comment:
This is the pylint I use on my team's Python projects. Edit it if you feel strongly about any of these rules!